docs/auth0_logs_tail.md

```diff
@@ -22,15 +22,15 @@ auth0 logs tail [flags]
   auth0 logs tail --filter "user_name:<user-name>"
   auth0 logs tail --filter "ip:<ip>"
   auth0 logs tail --filter "type:f" # See the full list of type codes at https://auth0.com/docs/logs/log-event-type-codes
-  auth0 logs tail -n 100
+  auth0 logs tail -n 10
```

Comment:
When tail'ing, its not as useful to initially print out a large number of requests. The output will continue to append, so its not as if this is the effective size of display. In fact, I doubt most users would want to tail out a particular number of logs.